pkg/sdk/testint/streams_gen_integration_test.go

```diff
@@ -0,0 +1,426 @@
+package testint
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_Streams(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	db, cleanupDb := createDatabase(t, client)
```

Comment:
You can already reuse one database, you can do it now or in following PR, up to you

Reply:
I'll do it in the next pr